src/Sulu/Bundle/ResourceBundle/Resource/FilterManager.php

```diff
@@ -467,8 +507,28 @@ public function getFeaturesForContext($context)
     }
 
     /**
+     * Removes conditions from condition groups when they are not in the given array
+     *
+     * @param ConditionGroupEntity $conditionGroup
+     * @param array $conditionIds
+     */
+    protected function removeNonExistentConditions(
+        $conditionGroup,
+        $conditionIds
```

Comment:
Why not passing the conditions to remove here and name the method `removeConditions`? Would be easier to understand.